azext_iot/common/certops.py

```diff
@@ -76,11 +96,13 @@ def open_certificate(certificate_path):
         certificate (str): returns utf-8 encoded value from certificate file.
     """
     certificate = ""
-    if certificate_path.endswith('.pem') or certificate_path.endswith('.cer'):
+    if certificate_path.endswith(".pem") or certificate_path.endswith(".cer"):
```

Comment:
Something we should probably patch up here while we're at it - if `certificate_path` is not a `.pem` or `.cer` file - we end up returning `""` for certificate here - should we instead throw an exception?

Reply:
.cert and .crt (as well as .cer) are common synonyms for .pem